### ISSUE TITLE
The tool has a slew of accessibility issues that should be addressed

### ISSUE THREAD

**Issue by KameronKerger** [Environmental-Policy-Innovation-Center/water-data-tool#128]
Running an [accessibility audit](https://www.accessibilitychecker.org/audit/?website=https%3A%2F%2Fapps.cnt.org%2Fwater-data-tool%2F&flag=us) on the tool reveals a myriad of accessibility issues including 43 critical ones, I believe on just the map page. Issues in the code structure should be addressed per the attached audit.

[apps.cnt.org-04.27.2026.pdf](https://github.com/user-attachments/files/27142891/apps.cnt.org-04.27.2026.pdf)

Definition of done:
- [ ] Issues in the code impacting the accessibility of the tool are addressed

**Comment by Gabe-Epic** [comment]
Needs a re-run on our hosted version @CSheesley
